data/ecosystems/m/moonwell.toml

```diff
@@ -30,14 +30,11 @@ url = "https://github.com/moonwell-fi/moonwell-subgraph"
 [[repo]]
 url = "https://github.com/moonwell-fi/moonwell.js"
 
-[[repo]]
-url = "https://github.com/moonwell-fi/nomad-agents"
-
 [[repo]]
 url = "https://github.com/moonwell-fi/proposal-verification"
 
 [[repo]]
 url = "https://github.com/moonwell-fi/temporal-governance"
 
 [[repo]]
-url = "https://github.com/moonwell-fi/yield-server"
+url = "https://github.com/moonwell-fi/yield-server"
```

Comment:
I'm seeing some changes that are going to conflict with your other PR (https://github.com/electric-capital/crypto-ecosystems/pull/613)